CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
     - [Server Binaries](#server-binaries)
   - [Changelog since v1.5.0](#changelog-since-v150)
     - [Other notable changes](#other-notable-changes)
+  - [Known Issues for v1.5.1](#known-issues-for-v151)
```

Comment:
I made this change intentionally, because otherwise in every future release we'll need to update all known issues links from `#known-issues-n` to be `#known-issues-n+1`. WDYT?